Content.Client/Access/UI/AccessLevelControl.xaml.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+using Content.Shared.Access;
+using Content.Shared.Access.Systems;
+
+namespace Content.Client.Access.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class AccessLevelControl : GridContainer
```

Comment:
wont require you to, but can you quickly make the door configurator or whatever that other PR added use this instead of copypasting?, that got past reviews